src/tests/node-filesystem.ts

```diff
@@ -1,6 +1,7 @@
 import { read, stat, open, ftruncate } from "fs";
 
 import { IFileSystem } from "@addon-types";
+// eslint-disable-next-line node/no-unpublished-import
```

Comment:
Ah, I see. We probably have this excluded in eslint config for tests in other projects.